.gitattributes

```diff
@@ -0,0 +1,2 @@
+**/config/crd/bases/**/*.yaml linguist-generated=true
```

Comment:
Q: Shouldn't this be:
`**/config/crd/bases/*.yaml`

cf. `[cluster-api](https://github.com/kubernetes-sigs/cluster-api)/[config](https://github.com/kubernetes-sigs/cluster-api/tree/main/config)/[crd](https://github.com/kubernetes-sigs/cluster-api/tree/main/config/crd)/bases/`

Reply:
Both work - but yours is more limited i.e. doesn't include subfolders which is more explicit and better in this case - I'll update.